backend/src/apiserver/resource/client_manager_fake.go

```diff
@@ -23,12 +23,6 @@ import (
 	"github.com/kubeflow/pipelines/backend/src/common/util"
 )
 
-const (
```

Comment:
Why remove the constants and replace them with hard-coded values?

Reply:
I did not remove them from the package, but rather moved to the corresponding `*_test.go` files. This will make sure that they are not included during compilation. These constant should not be exported as they are used for testing, while the types defined in the file should. Using hard-coded values in unit-tests may be more robust and is a better practice as tests should be self-contained.